eng/Versions.props

```diff
@@ -42,7 +42,7 @@
     <SystemTextJsonVersion>7.0.3</SystemTextJsonVersion>
     <SystemReflectionMetadataLoadContextVersion>9.0.0-alpha.1.23516.25</SystemReflectionMetadataLoadContextVersion>
     <SystemManagementPackageVersion>4.6.0</SystemManagementPackageVersion>
-    <SystemCommandLineVersion>2.0.0-beta4.23307.1</SystemCommandLineVersion>
+    <SystemCommandLineVersion>2.0.0-beta4.23326.1</SystemCommandLineVersion>
```

Comment:
This PR is introducing a type of incoherency which is causing source-build prebuilds.  This version does not match the version specified in the Version.Details.xml file.

Reply:
I've updated this version to align with the one in Version.Details.xml. Then, fixed any build failures. Thanks!